catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswExceptionMapper.java

```diff
@@ -35,11 +40,20 @@ public Response toResponse(Throwable exception) {
         if (exception instanceof CswException) {
             cswException = (CswException) exception;
         } else {
-            cswException = new CswException(
-                    "Error parsing the request.  XML parameters may be missing or invalid.",
-                    CswConstants.MISSING_PARAMETER_VALUE,
-                    null);
+            String message = exception.getMessage();
+            if (StringUtils.isBlank(message)) {
+                cswException = new CswException(
+                        "Error parsing the request.  XML parameters may be missing or invalid.",
+                        exception);
+            } else {
+                cswException = new CswException(
+                        "Error handling Request: " + message,
```

Comment:
Looking at the OWS schema, reporting this should be fine http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd. @stustison / @kcwire is there a concern of exposing an implementation detail here?